mbf_costmap_nav/scripts/move_base_legacy_relay.py

```diff
@@ -115,23 +115,25 @@ def mb_reconf_cb(config, level):
 
 
 if __name__ == '__main__':
-    rospy.init_node("move_base")
+    rospy.init_node("move_base_legacy_relay")
+    node_name = rospy.get_param('~move_base_flex_node_name', 'move_base_flex')
+    move_base_node_name = rospy.get_param('~move_base_node_name', 'move_base')
 
     # TODO what happens with malformed target goal???  FAILURE  or INVALID_POSE
     # txt must be:  "Aborting on goal because it was sent with an invalid quaternion"   
 
     # move_base_flex get_path and move_base action clients
-    mbf_mb_ac = actionlib.SimpleActionClient("move_base_flex/move_base", mbf_msgs.MoveBaseAction)
-    mbf_gp_ac = actionlib.SimpleActionClient("move_base_flex/get_path", mbf_msgs.GetPathAction)
+    mbf_mb_ac = actionlib.SimpleActionClient(node_name + "/move_base", mbf_msgs.MoveBaseAction)
+    mbf_gp_ac = actionlib.SimpleActionClient(node_name + "/get_path", mbf_msgs.GetPathAction)
```

Comment:
Would it be better to use the private namespace? What would you say?

Reply:
This node is for relaying from `move_base_flex` topics to `move_base`.
ant these node_names are not under this node's name space.
so we should not use the private namespace.

Reply:
Okay. Thank you for the clarification!